### ISSUE TITLE
container paths 

### ISSUE THREAD

**Issue by sergiobelotti** [rvilalta/ietf-te-path-computation#44]
Since the RPC path computation is a stateless RPC , there is no need to have a container storing computed paths information, and therefore also the path-id has to be deleted from the info provided as output of RPC.

```
container paths {
    list path {
      key "path-id";
      config false;
      uses path-info;
      description "List of previous computed paths.";
    }
    description "Root container for path-computation";
  }

grouping path-info {
    leaf **path-id** {
      type yang-types:uuid;
      config false;
      description "path-id ref.";
    }
    uses te-types:generic-path-properties;
    description "Path computation output information";
  }

```

**Comment by sergiobelotti** [comment]
The conclusion is to remove the grouping and related container.

**Comment by carloperocchio** [comment]
See commit https://github.com/rvilalta/ietf-te-path-computation/commit/bf42cf2a59d87c989c1a9bc34fb26dcffefb5671

**Comment by italobusi** [comment]
The bug has been closed with commit bf42cf2
